### DISCUSSION TITLE
헤더 이름이 다른건가요?

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#113] [Q&A] [answered]
1
---
![image](https://user-images.githubusercontent.com/62102830/203547636-509a7123-0008-400c-8b44-c77bdb166917.png)

스크럼때 포함시켜달라는 파일 중 my_string_tests.h는 없었습니다.

my_string_test_cases.h로 수정하였습니다.

2
---
![image](https://user-images.githubusercontent.com/62102830/203549305-bd12b8f9-954b-498f-90dd-f32bf751e730.png)

size_t에 에러가 잡혀 해당 헤더 파일에 <iostream>을 추가하였습니다.

**Comment by Woon-2** [comment, marked as answer]
1. `my_string_test_cases.h` 맞습니다.   
2. `<iostream>`을 `include`하면 될 것 같네요.
